react-sdk/CHANGELOG.md

```diff
@@ -9,134 +9,115 @@
 
```

Comment:
Also still a CHANGELOG.md in here

Reply:
How about now?

Reply:
The changelog file contents are exactly the same as in my PR not sure why the diff is showing something else.